app/os/main.cpp

```diff
@@ -56,6 +56,7 @@
 #include "RobotController.h"
 #include "SDBlockDevice.h"
 #include "SerialNumberKit.h"
+#include "ShapeRecognition.h"
```

Comment:
Could we have a header for all autonomous activities in order to avoid to overcrowd  the included files here?

```suggestion
#include "AutonomousActivities.h"
```
```suggestion
#include "Activities.h"
```